app/src/organisms/GripperWizardFlows/index.tsx

```diff
@@ -77,24 +91,31 @@ export function GripperWizardFlows(
 
   const handleCleanUpAndClose = (): void => {
     setIsExiting(true)
-    chainRunCommands([{ commandType: 'home' as const, params: {} }], true)
-      .then(() => {
-        deleteMaintenanceRun(maintenanceRunId)
-        setIsExiting(false)
-        props.onComplete?.()
-      })
-      .catch(error => {
-        console.error(error.message)
-        deleteMaintenanceRun(maintenanceRunId)
-        setIsExiting(false)
-        props.onComplete?.()
-      })
+    if (maintenanceRunData?.data.id == null) closeFlow()
+    else {
+      chainRunCommands(
```

Comment:
Probably using {} would make code more readable.

```suggestion
   if (maintenanceRunData?.data.id == null) {
      closeFlow()
    } else {
```